sdk/containerregistry/azure-containerregistry/azure/containerregistry/_models.py

```diff
@@ -132,7 +132,6 @@ class RepositoryProperties(object):
     :vartype last_updated_on: :class:`datetime.datetime`
     :ivar int manifest_count: Number of manifest in the repository
     :ivar str name: Name of the repository
-    :ivar str registry: Registry the repository belongs to
```

Comment:
We don't think this would be useful info?

Reply:
This was an error on my part, the registry is not included in the response object.